recipes-devtools/firmware-package-tool/firmware-package-tool.bb

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+DESCRIPTION = "Generate The Firmware Update Package"
+MAINTAINER = "huaqian.li@siemens.com"
+
+SRC_URI = "file://update.conf.json \
+           file://iot2050-fwu-update-json.sh"
+
+inherit dpkg-raw
```

Comment:
You are not building a deb package, are you?

Reply:
Yes, I didn't find a more elegant way. If you know, please suggest me, thanks.

Reply:
If you are not building a deb package, don't inherit a class that tries to do that. But as I said, this path is not the best one anyway.